docs/source/apidoc.md

```diff
@@ -134,6 +134,16 @@ See Ref. :cite:`Czarnik_2021_Quantum` for more details on these methods.
    :members:
 ```
 
+## Classical Shadows
+
```

Comment:
Please add a one-line description of the package, like it's done for the other techniques. You could include references too.

Reply:
sorry, I didn't find any descriptions on the apidoc for any packages (i.e., headers of `##`), so I find no examples there. But I do see some one-line descriptions under `###` headers. I'll add necessary descriptions to the header of files.